client/browser/package.json

```diff
@@ -4,31 +4,31 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "engines": {
-    "yarn": ">1.10.0"
+    "pnpm": ">=7"
   },
   "scripts": {
-    "storybook": "STORIES_GLOB='client/browser/src/**/*.story.tsx' yarn workspace @sourcegraph/storybook run start",
+    "storybook": "STORIES_GLOB='client/browser/src/**/*.story.tsx' pnpm -F @sourcegraph/storybook start",
     "build-inline-extensions": "node scripts/build-inline-extensions",
-    "predev": "yarn run build-inline-extensions",
-    "dev": "yarn run predev && NODE_ENV=development NODE_OPTIONS=--max_old_space_size=8192 node -r ts-node/register scripts/development",
-    "dev:no-reload": "AUTO_RELOAD=false yarn run dev",
-    "dev:firefox": "if type web-ext 2>/dev/null; then yarn dev & web-ext run --source-dir ./build/firefox; else echo 'web-ext not found. Install it with: yarn global add web-ext'; exit 1; fi",
-    "dev:chrome": "TARGETS=chrome yarn run dev",
-    "dev:edge": "TARGETS=edge yarn run dev",
-    "dev:safari": "TARGETS=safari yarn run build",
-    "prebuild": "yarn -T run generate && yarn run build-inline-extensions",
```

Comment:
This `yarn -T` is an example where this script must run in the root. The `generate` is in the root package.json, trying to fix this now...